github/messages.go

```diff
@@ -30,6 +31,35 @@ const (
 	sha512Prefix = "sha512"
 	// signatureHeader is the GitHub header key used to pass the HMAC hexdigest.
 	signatureHeader = "X-Hub-Signature"
+	// eventTypeHeader is the Github header key used to pass the event type
+	eventTypeHeader = "X-Github-Event"
+)
+
+var (
+	// eventTypeMapping is a conversion map between go-github types and webhooks types
```

Comment:
Also missing a period.

The phrasing is odd. Isn't it a map from webhook types (e.g., "commit_comment") and go-github types (e.g., `CommitCommentEvent`)? The current phrasing makes it sound like the inverse of that.